monocular_pose_estimator_lib/CMakeLists.txt

```diff
@@ -3,21 +3,24 @@ project(monocular_pose_estimator_lib)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O3")
 
-find_package(catkin REQUIRED COMPONENTS cmake_modules roscpp)
-find_package(Eigen REQUIRED)
+find_package(catkin REQUIRED COMPONENTS roscpp)
 find_package(OpenCV REQUIRED)
```

Comment:
OpenCV install an OpenCVConfig.cmake in /usr/share/OpenCV which has this `set(OpenCV_LIBRARIES ${OpenCV_LIBS})` so both are OK. I changes to `LIBRARIES` to follow catkin conventions.